strangerstudios/paid-memberships-pro
# Single level block

### All Submissions:






### Changes proposed in this Pull Request:

Clarifying issue where block frontend rendering issue 

Resolves XXX.

### How to test the changes in this Pull Request:

1. Open an empty WordPress page
2. Select single level block 
3. Verify if block renders correctly on the frontend

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

- ISSUE CLARIFICATION: Finding issue where parent block will not render children blocks correctly on the frontend.
